doc/api/packages.md

```diff
@@ -872,6 +872,10 @@ changes:
     - v13.7.0
     pr-url: https://github.com/nodejs/node/pull/31008
     description: Implement logical conditional exports ordering.
+  - version:
+    - REPLACEME
+    pr-url: https://github.com/nodejs/node/pull/34718
+    description: Add support for `"exports"` patterns.
```

Comment:
Fwiw, newer changes are usually listed first (although the documentation tooling should sort these anyway)

Reply:
Since the tooling indeed re-order the items, the most important is probably to ensure the changes follow the same convention as the rest of the file, right?

Not entirely related, but for version numbers (in case of backport to LTS lines), the older version comes first (and the tooling doesn't re-order those).

https://github.com/nodejs/node/blob/1390574d66c1be8da899d18fe09afc5cc568fb79/doc/api/packages.md#L860-L862

Reply:
> Since the tooling indeed re-order the items, the most important is probably to ensure the changes follow the same convention as the rest of the file, right?

If this file differs from the convention for the rest of the docs, I’d say we should also change this for this file :)

Reply:
I dig a little more into it, currently there is:

- 149 YAML comments that follow the `packages.md` convention.
- 237 YAML comments that follow the `fs.md` convention.

You're correct that the majority of the docs use a different convention, but clearly both are in used today. That'd be great to enforce one or the other for the sake of consistency, but I don't think it should be done in this PR.

Reply:
This is done in https://github.com/nodejs/node/pull/35454.